kernel/Dockerfile

```diff
@@ -50,13 +53,27 @@ RUN curl -fsSLO ${KERNEL_SHA256_SUMS} && \
     gpg2 --verify linux-${KERNEL_VERSION}.tar.sign linux-${KERNEL_VERSION}.tar && \
     cat linux-${KERNEL_VERSION}.tar | tar --absolute-names -x && mv /linux-${KERNEL_VERSION} /linux
 
-COPY kernel_config-${KERNEL_SERIES} /linux/arch/x86/configs/x86_64_defconfig
+# When using COPY with more than one source file, the destination must be a directory and end with a /
+COPY kernel_config-${KERNEL_SERIES}* /linux/
 COPY kernel_config.debug /linux/debug_config
 
-RUN if [ -n "${DEBUG}" ]; then \
+RUN case $(uname -m) in \
+    x86_64) \
+    cp /linux/kernel_config-${KERNEL_SERIES} /linux/arch/x86/configs/x86_64_defconfig; \
+    if [ -n "${DEBUG}" ]; then \
     sed -i 's/CONFIG_PANIC_ON_OOPS=y/# CONFIG_PANIC_ON_OOPS is not set/' /linux/arch/x86/configs/x86_64_defconfig; \
     cat /linux/debug_config >> /linux/arch/x86/configs/x86_64_defconfig; \
-    fi
+    fi \
+    ;; \
+    aarch64) \
+    cp /linux/kernel_config-${KERNEL_SERIES}-aarch64 /linux/arch/arm64/configs/defconfig; \
+    if [ -n "${DEBUG}" ]; then \
+    sed -i 's/CONFIG_PANIC_ON_OOPS=y/# CONFIG_PANIC_ON_OOPS is not set/' /linux/arch/arm64/configs/defconfig; \
+    cat /linux/debug_config >> /linux/arch/arm64/configs/defconfig; \
+    fi \
```

Comment:
there is a fair bit of replication here. It might be easier and simpler if the `case` statement just copies the config file and sets an env variable. Then the rest of the code is shared